tests/tck/features/match/SameTagPropname.feature

```diff
@@ -0,0 +1,212 @@
+      | "p_qian"      | ("qian" :player{age: 19, height: 179.0, name: "p_qian"} :student{age: 19, name: "s_qian", score: 88.0})    |
+      | "p_zhao"      | ("zhao" :player{age: 20, height: 187.0, name: "p_zhao"} :student{age: 20, name: "s_zhao", score: 99.0})    |
+      | "p_wang"      | ("wang" :player{age: 22, height: 192.0, name: "p_wang"} :student{age: 22, name: "s_wang", score: 82.0})    |
+      | "p_li"        | ("li" :player{age: 20, height: 201.0, name: "p_li"} :student{age: 20, name: "s_li", score: 91.0})          |
+      | "p_zhang"     | ("zhang" :player{age: 18, height: 189.0, name: "p_zhang"} :student{age: 18, name: "s_zhang", score: 89.0}) |
+      | "p_sun"       | ("sun" :player{age: 17, height: 192.0, name: "p_sun"} :student{age: 17, name: "s_sun", score: 72.0})       |
+    When executing query:
+      """
+      match (v:player) where v.player.age > 20 return v.player.name, v
+      """
+    Then the result should be, in any order:
+      | v.player.name | v                                                                                                       |
+      | "p_wang"      | ("wang" :player{age: 22, height: 192.0, name: "p_wang"} :student{age: 22, name: "s_wang", score: 82.0}) |
+    When executing query:
+      """
+      match (v:student)  return v.student.name, v
+      """
+    Then the result should be, in any order:
+      | v.student.name | v                                                                                                          |
+      | "s_qian"       | ("qian" :player{age: 19, height: 179.0, name: "p_qian"} :student{age: 19, name: "s_qian", score: 88.0})    |
+      | "s_zhao"       | ("zhao" :player{age: 20, height: 187.0, name: "p_zhao"} :student{age: 20, name: "s_zhao", score: 99.0})    |
+      | "s_wang"       | ("wang" :player{age: 22, height: 192.0, name: "p_wang"} :student{age: 22, name: "s_wang", score: 82.0})    |
+      | "s_li"         | ("li" :player{age: 20, height: 201.0, name: "p_li"} :student{age: 20, name: "s_li", score: 91.0})          |
+      | "s_zhang"      | ("zhang" :player{age: 18, height: 189.0, name: "p_zhang"} :student{age: 18, name: "s_zhang", score: 89.0}) |
+      | "s_sun"        | ("sun" :player{age: 17, height: 192.0, name: "p_sun"} :student{age: 17, name: "s_sun", score: 72.0})       |
+    When executing query:
+      """
+      match (v:student) where v.student.age > 20 return v.student.name, v
+      """
+    Then the result should be, in any order:
+      | v.student.name | v                                                                                                       |
+      | "s_wang"       | ("wang" :player{age: 22, height: 192.0, name: "p_wang"} :student{age: 22, name: "s_wang", score: 82.0}) |
+    When executing query:
+      """
+      match (v:player)-[e:like]->(d) where v.player.age > 19 return e, v.student.name, v.player.name, v.name
```

Comment:
I don't know whether we need to keep the compatibility with older version. My suggest is to report the error to user.

Reply:
Supporting both expressions `v.tag.prop` and `v.prop` at the same time is confusing.

Reply:
so what's the decision?

Reply:
not compatible with the previous version,will modify test case